### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Deploy to Azure_Keyboard: Information available from the "i" tooltip is not available for keyboard only users

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3347]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.1.1

**Repro Steps:**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, right-click your project, expand Azure, then click Deploy to Azure
11) Verify if the information available from the "i" tooltip is available for keyboard only users

**Actual Result:**
Information available from the "i" tooltip is not available for keyboard only users

**Expected Result:**
Information available from the "i" tooltip should also be available for keyboard only users

**User Impact:**
Keyboard only users will not be provided with the information present in "i"

**MAS Reference:**
[MAS 2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

[Bug_9_IntelliJ_Deploy to Azure_i_Keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451325/Bug_9_IntelliJ_Deploy.to.Azure_i_Keyboard.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram This issue has been fixed in #3525, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1245/Azure/), please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
Verified this issue in the build mentioned above. This issue is fixed. Hence closing this. Added attachments for reference.
[Bug_3347_i_Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3621041/Bug_3347_i_Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749762
